com.ibm.streamsx.inet/com.ibm.streamsx.inet/InetSource/InetSource_h.cgt

```diff
@@ -125,8 +125,19 @@ private:
   int32_t iteration_;
   uint32_t emitTuplePerRecordCount_;
   bool dynamicURL_;
-  MY_OPERATOR(const MY_OPERATOR& op);
-  MY_OPERATOR& operator=(const MY_OPERATOR& op);
+
+  // ----------- operator output tuple ----------
+
+  OPort0Type outputTuple;
+
+  // ----------- assignment functions for output attributes ----------
+
+  SPL::rstring TARGET_URL() { return retriever_->targetURL(); }
```

Comment:
I don't think these functions should be in the SPL namespace, that's owned by the product.

How about `streamsx::inet`